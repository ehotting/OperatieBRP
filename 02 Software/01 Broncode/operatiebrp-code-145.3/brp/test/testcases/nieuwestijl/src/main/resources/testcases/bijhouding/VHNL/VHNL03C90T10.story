Meta:
@auteur                 tjlee
@status                 Klaar
@regels                 R2167
@sleutelwoorden         voltrekkingHuwelijkInNederland,registratieNaamgebruik,TjieWah,VHNL03C90T10
@usecase                UCS-BY.HG

Narrative:
R2167 Huwelijk en geregistreerd partnerschap, voltrekking huwelijk in Nederland, actie registratieAanvangHuwelijkGeregistreerdPartnerschap, registratieNaamgebruik

Scenario: R2167 Persoon.Naamgebruik afgeleid is niet ingevuld
          LT: VHNL03C90T10

Gemeente BRP 1

Given alle personen zijn verwijderd

Given enkel initiele vulling uit bestand /LO3PL/VHNL03C90-sandy.xls
Given enkel initiele vulling uit bestand /LO3PL/VHNL03C90-danny.xls

When voer een bijhouding uit VHNL03C90T10.xml namens partij 'Gemeente Tiel'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL03C90T10.txt

Then lees persoon met anummer 9576485650 uit database en vergelijk met expected VHNL03C90T10-persoon1.xml
Then lees persoon met anummer 3604914962 uit database en vergelijk met expected VHNL03C90T10-persoon2.xml
