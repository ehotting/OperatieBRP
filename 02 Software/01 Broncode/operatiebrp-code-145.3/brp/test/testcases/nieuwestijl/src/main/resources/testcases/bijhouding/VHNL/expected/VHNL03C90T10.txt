cvc-complex-type.2.4.b: The content of element 'brp:naamgebruik' is not complete. One of '{"http://www.bzk.nl/brp/brp0200":indicatieAfgeleid}' is expected.